lib/ansible/modules/gather_facts.py

```diff
@@ -13,7 +13,9 @@
 version_added: 2.8
 short_description: Gathers facts about remote hosts
 description:
-     - This module takes care of executing the configured facts modules, the default is to use the M(ansible.builtin.setup) module.
+     - This module takes care of executing the
+       L(configured facts modules,https://docs.ansible.com/ansible/latest/reference_appendices/config.html#facts-modules),
```

Comment:
This probably should use some internal ref, instead of absolute linking.  I'm just not an expert on what that would look like.  Maybe:

```
:ref:`FACTS_MODULES`
```

Reply:
I thought so too, just couldn't figure out how. I'll post in IRC and see if anyone has ideas.

Reply:
@sivel's example looks correct. I have a config.rst floating around in my ansible tree (I guess built by some part of the docs build), which contains:
```.rst
.. _FACTS_MODULES:

FACTS_MODULES
-------------
```

Reply:
Well, at least the reference is correct. The RST syntax is not valid in module docs. `R(Put some title here, FACTS_MODULES)` seems to be the correct syntax (https://github.com/ansible-community/antsibull/blob/main/antsibull/jinja2/filters.py#L20).

Reply:
(Will obviously not be very helpful in ansible-doc text output, though.)

Reply:
its also missing reference to the variables that can override that